api/room/models.py

```diff
@@ -21,6 +21,7 @@ class Room(Base, Utility):
     image_url = Column(String)
     calendar_id = Column(String)
     floor_id = Column(Integer, ForeignKey('floors.id'))
+    tag_id = Column(Integer, ForeignKey('tags.id'))
```

Comment:
Is it possible to have one room having more than one tag?

Reply:
Yes, a room can have more than one tag. This is done in the room_tags table where several room id's can have the same tag id.
Looking at this again, I see there is no need to have a tag_id in the room models. This will be removed shortly.

Reply:
The tag id has now been removed from the room model.